[2024-10-28T21:22:26.098+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-28T21:22:26.354+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tab_refresh_stats.daily_instagram_stats manual__2024-10-28T21:21:59.558146+00:00 [queued]>
[2024-10-28T21:22:26.359+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tab_refresh_stats.daily_instagram_stats manual__2024-10-28T21:21:59.558146+00:00 [queued]>
[2024-10-28T21:22:26.359+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-10-28T21:22:26.378+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): daily_instagram_stats> on 2024-10-28 21:21:59.558146+00:00
[2024-10-28T21:22:26.391+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=768) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-28T21:22:26.392+0000] {standard_task_runner.py:72} INFO - Started process 770 to run task
[2024-10-28T21:22:26.393+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'tab_refresh_stats', 'daily_instagram_stats', 'manual__2024-10-28T21:21:59.558146+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/tab_social_media_refresh.py', '--cfg-path', '/tmp/tmpgjm66t4h']
[2024-10-28T21:22:26.395+0000] {standard_task_runner.py:105} INFO - Job 29: Subtask daily_instagram_stats
[2024-10-28T21:22:26.432+0000] {task_command.py:467} INFO - Running <TaskInstance: tab_refresh_stats.daily_instagram_stats manual__2024-10-28T21:21:59.558146+00:00 [running]> on host 721aaaeb5415
[2024-10-28T21:22:26.506+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='daniel.cristancho' AIRFLOW_CTX_DAG_ID='tab_refresh_stats' AIRFLOW_CTX_TASK_ID='daily_instagram_stats' AIRFLOW_CTX_EXECUTION_DATE='2024-10-28T21:21:59.558146+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-28T21:21:59.558146+00:00'
[2024-10-28T21:22:26.507+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-28T21:22:28.199+0000] {evolution_stats_instagram.py:35} INFO - Start extract loaded data
[2024-10-28T21:22:34.452+0000] {evolution_stats_instagram.py:38} INFO - Start load new data
[2024-10-28T21:22:34.452+0000] {evolution_stats_instagram.py:108} INFO - writing user_follow_stats table
[2024-10-28T21:22:36.758+0000] {evolution_stats_instagram.py:112} INFO - 5 rows load in user_follow_stats
[2024-10-28T21:22:36.758+0000] {evolution_stats_instagram.py:113} INFO - loading operation finished successfully for user_follow_stats
[2024-10-28T21:22:36.759+0000] {logging_mixin.py:190} INFO - ****************** Success process to update instagram post
[2024-10-28T21:22:36.759+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-28T21:22:36.769+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-28T21:22:36.769+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=tab_refresh_stats, task_id=daily_instagram_stats, run_id=manual__2024-10-28T21:21:59.558146+00:00, execution_date=20241028T212159, start_date=20241028T212226, end_date=20241028T212236
[2024-10-28T21:22:36.796+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-28T21:22:36.815+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-28T21:22:36.817+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
